cluster/manifests/admission-control/teapot.yaml

```diff
@@ -0,0 +1,29 @@
+{{ if eq .ConfigItems.teapot_webhook_enabled "true" }}
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: MutatingWebhookConfiguration
+metadata:
+  name: teapot-admission-controller
+  labels:
+    application: teapot-admission-controller
+webhooks:
+  - name: pod-admitter.teapot.zalan.do
+    clientConfig:
+      url: "https://localhost:8085/pod"
+      caBundle: "{{ .ConfigItems.ca_cert_decompressed }}"
+    failurePolicy: Fail
```

Comment:
Does this tell the API server what to do in case of this webhook being unavailable? Isn't it safer to allow Pod creation in those cases? Yes, they could circumentvent the (non security-relevant) restrictions but all pods get re-created on the next update anyways.

Reply:
I'd rather ensure that it's always available. If we start putting more helpful stuff into the webhook (e.g. TOKENINFO_URL injection), it's going to be weird for the users to suddenly get pods created without that. Plus the controller would retry pod creation again soon.

Reply:
Ok, I didn't think about the retries.